### PR TITLE
Test administrator

### PR DESCRIPTION
**Pull request by vadymtsehelny**

### CODE REVIEW

File: src/test/java/com/easyrest/tests/BaseTest.java [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.testng.annotations.AfterTest;
-import org.testng.annotations.BeforeTest;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.io.File;
 import java.io.IOException;
 import java.time.Duration;
-import com.aventstack.extentreports.ExtentReports;
```
Comment (Zeicks) [inline src/test/java/com/easyrest/tests/BaseTest.java]: Are you sure that by deleting all this import this you are not breaking anything?

File: src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import org.testng.annotations.Test;
+
+import static com.easyrest.config.ConfigProvider.administratorEmail;
+import static com.easyrest.config.ConfigProvider.administratorPassword;
+import static com.easyrest.constants.Constants.AdministratorTabMenu.*;
+
+public class AdministratorTest extends BaseTest {
+    private AdministratorOperationsFacade administrator;
+
+    @BeforeMethod
```
Comment (martadiachok) [inline src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java]: why do you need to login and log out after each test?

File: src/main/java/com/easyrest/constants/Constants.java [hunk 3]
```diff
@@ -6,4 +6,12 @@ public static class TimeoutVariable {
         public static final int IMPLICIT_WAIT = 4;
         public static final int EXPLICIT_WAIT = 7;
     }
+
+    public static class AdministratorTabMenu {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/constants/Constants.java:10]: will you use this constant in one class or in several classes?

File: src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java (hunk 2 above)
Comment (vadymtsehelny) [reply src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java]: each test should be independent of any other test.

File: src/test/java/com/easyrest/tests/BaseTest.java (hunk 1 above)
Comment (vadymtsehelny) [reply src/test/java/com/easyrest/tests/BaseTest.java]: It was code formatting, these lines moved up.

File: src/main/java/com/easyrest/constants/Constants.java (hunk 3 above)
Comment (vadymtsehelny) [reply src/main/java/com/easyrest/constants/Constants.java:10]: I use this constant in one class for readability and store them in a constants package for maintainability.

**Review by Zeicks — APPROVED** [review]

File: src/main/java/com/easyrest/components/administrator/AdministratorMenu.java [hunk 4]
```diff
@@ -34,4 +35,8 @@ public void clickOnWaiters() {
         driver.findElement(waitersOrdersTab).click();
     }
 
+    public String seeWhichTabIsOpen() {
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/administrator/AdministratorMenu.java]: Please rename the method, e.g. getSelectedTabName()

Comment (vadymtsehelny) [reply src/main/java/com/easyrest/components/administrator/AdministratorMenu.java]: Done

File: src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java (hunk 2 above)
Comment (martadiachok) [reply src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java]: > each test should be independent of any other test.

but if first test fails the program will execute second test. If you do login and log out after each test there is more time execution.

**Review by dmatsev — APPROVED** [review]

**Pull request merged**